### PR TITLE
Galaxy counts

### PR DESCRIPTION
**Pull request by rhiannonlynne**

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
Might as well see what code review looks like on github.

File: mafContrib/lssMetrics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class GalaxyCountsMetric(BaseMetric):
+    """
+    Estimate the number of galaxies expected at a particular coadded depth.
+    """
+    def __init__(self, m5Col = 'fiveSigmaDepth', nside=128, metricName='GalaxyCounts', **kwargs):
+        self.m5Col = m5Col
+        super(GalaxyCountsMetric, self).__init__(col=self.m5Col, metricName=metricName, **kwargs)
+        # Use the coadded depth metric to calculate the coadded depth at each point.
```
Comment (yoachim) [inline mafContrib/lssMetrics.py]: need to set self.units so it doesn't auto-set to mags.

File: mafContrib/lssMetrics.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    def run(self, dataSlice, slicePoint=None):
+        # Calculate the coadded depth.
+        coaddm5 = self.coaddmetric.run(dataSlice)
+        # Calculate the number of galaxies.
+        # From Carroll et al, 2014 SPIE (http://arxiv.org/abs/1501.04733)
+        # Instead of a number of galaxies accurate on an absolute scale,
+        #  this may give the number of galaxies on a relative scale as I haven't
+        #  included the effects of a rollover in efficiency around the m5 value,
+        #  or the size of the healpix, or account for the overall number of galaxies around the sky.
+        num_gal, intErr = scipy.integrate.quad(self._galCount, -np.inf, 32, args=coaddm5)
```
Comment (yoachim) [inline mafContrib/lssMetrics.py]: The comment seems out-of-date now.

**Comment by yoachim** [comment]
hmm, there's no handy "Approve" button like Stash.  Am I just supposed to comment "Approved", or should I close the pull request as well?

**Comment by rhiannonlynne** [comment]
I think when we're "done", we are supposed to add a milestone or something? Or maybe I was supposed to open an associated issue? Hmm, not sure. Saying it's done is fine for me for now. I guess not closing the branch is good, since there are small issues to update.

**Comment by drphilmarshall** [comment]
I was just about to suggest you merge this. If there are remaining small
issues, you should start an issue on each of them! Milestones are things
you define in advance, so you can assign issues to them: when all of a
milestone's issues are closed, and you have no more to make, you can close
the milestone. They're for keeping projects on track. A good system for
milestones is to figure out what your next delivery date is, and make a
milestone for that. The next cadence workshop should definitely be a
milestone, but there's probably going to be some before that. Any sort of
release coming up?

**Pull request merged**